src/xo-app/host/tab-patches.js

```diff
@@ -111,6 +115,34 @@ const INSTALLED_PATCH_COLUMNS_2 = [
   needsRestart: createDoesHostNeedRestart((_, props) => props.host)
 }))
 export default class HostPatches extends Component {
+  _installPatchWarning = ({patch, installPatch}) => confirm(
+    {
+      title: _('installPatchWarningTitle'),
+      body: <div>
+        <p>{_('installPatchWarningContent')}</p>
+        <Link to={'/pools/' + this.props.host.$pool + '/patches'}>
+          <Button
+            btnStyle='success btn-block'
+            onClick={close}>Go to pool level patching
+          </Button>
+        </Link>
+      </div>
+    }).then(() => installPatch(patch))
+
+  _installAllPatchesWarning = installAllPatches => confirm(
+    {
+      title: _('installPatchWarningTitle'),
+      body: <div>
+        <p>{_('installPatchWarningContent')}</p>
+        <Link to={'/pools/' + this.props.host.$pool + '/patches'}>
+          <Button
+            btnStyle='success btn-block'
+            onClick={close}>Go to pool level patching
+          </Button>
+        </Link>
+      </div>
+    }).then(() => installAllPatches())
+
```

Comment:
If necessary, you can create a `ButtonLink` component, based on `Button` which calls [`router.push()`](https://github.com/ReactTraining/react-router/blob/v3/docs/API.md#pushpathorloc).